src/webcomponents/helperfunctions/jsonToCol.ts

```diff
@@ -0,0 +1,51 @@
+import { JsonToCol, LanguagesInput } from "../../@types/interfaces";
+
+const mapLangToSheetColumns = (languages: LanguagesInput) => {
+  let columnTitle: Array<object> = [];
+  Object.keys(languages).map(
+    (title: string) =>
+      (columnTitle = [
+        ...columnTitle,
+        { type: "text", title: `${title}`, width: 150 },
+      ])
+  );
+  return columnTitle;
+};
+
+const setKeys = (languages: LanguagesInput): Array<string> => {
+  let languagesKeys: Array<string> = [];
```

Comment:
use `new Set()`

try to use `[].reduce()` and add all `messageID`s to the `Set`

Reply:
where should i use new set()?

Reply:
Did It